live/global.ini

```
@@ -41925,20 +41925,20 @@ blackbox_recover_obj_short_03=BlackBox liefern
-blackboxillegal_desc_0004=Die Leute fangen an herumzuschnüffeln und versuchen herauszufinden, was mit dieser "~mission(Ship)" passiert ist. Das ist ein Problem für ~mission(Client). Sie würden es bevorzugen, wenn die Vergangenheit begraben bliebe. Das heißt, sie wären superglücklich, wenn sich jemand die "~mission(Item)" schnappen und ihn zu ~mission(Destination) bringen würde. Die Art von Freude, die sich gut bezahlt macht. ~mission(Timed)~mission(Danger)
-blackboxillegal_desc_0005=Ich suche nach einem moralisch flexiblen Typ, der losgeht und eine "~mission(Item)" von einer "~mission(Ship)" abholt, die havariert ist. Wir senden ~mission(Location), Du musst also nur dorthin gehen, es sich schnappen und es an ~mission(Destination) abgeben. Einfach. ~mission(Danger)~mission(Timed)
-blackboxillegal_desc_0006=Wir alle tun Dinge, die wir irgendwann mal bereuen, oder? Es scheint, dass ~mission(Client) etwas übertrieben hat und eine "~mission(Ship)" abstürzen ließ. Jetzt möchten sie, dass jemand zu ~mission(Location) geht und die "~mission(Item)" holt, bevor dich jemand findet. Dann müsst du es nur noch zu ~mission(Destination) bringen und bezahlt werden. ~mission(Timed)~mission(Danger)
+blackboxillegal_desc_0001=Eine "~mission(Ship)" wurde in die Luft gesprengt und es heißt, dass ~mission(Client) möglicherweise etwas damit zu tun hat. Sie lassen ein paar Scheine dafür springen, wenn du zu ~mission(Location) schippert, dir die "~mission(Item)" schnappst und zur Vernichtung zu ~mission(Destination) bringt. ~mission(Timed)~mission(Danger)
+blackboxillegal_desc_0002=Hier ist der Haken. Ich brauchen jemanden, der sich zu ~mission(Location) begibt und eine "~mission(Item)" abholt. ~mission(Client) zahlt viel Geld, um diesen "Unfall" zu verschleiern. Alles, was du tun musst, ist sie bei ~mission(Destination) abzugeben und dein Geld abzuholen. Ziemlich einfach also. ~mission(Timed)~mission(Danger)
+blackboxillegal_desc_0003=Eine "~mission(Ship)" hatsich selbst zerstört. Das Problem ist, dass die ~mission(Item) aufgezeichnet hat, wer die Tat begangen hat. Jetzt braucht ~mission(Client) jemanden, der sich zu ~mission(Location) begibt und die "~mission(Item)" nach ~mission(Destination) bringt. ~mission(Timed)~mission(Danger)
+blackboxillegal_desc_0004=Die Leute fangen an herumzuschnüffeln und versuchen herauszufinden, was mit der "~mission(Ship)" passiert ist. Für ~mission(Client) ist das ein ziemlich großes Problem und bevorzugt es, wenn die Vergangenheit begraben bliebe. Das heißt, es wäre großartig, wenn sich jemand die "~mission(Item)" schnappt und nach ~mission(Destination) bringt. Die Art von Freude, die sich gut bezahlt macht. ~mission(Timed)~mission(Danger)
```

```suggestion
blackboxillegal_desc_0004=Die Leute fangen an herumzuschnüffeln und versuchen herauszufinden, was mit der "~mission(Ship)" passiert ist. Für ~mission(Client) ist das ein ziemlich großes Problem und er bevorzugt es, wenn die Vergangenheit begraben bliebe. Das heißt, es wäre großartig, wenn sich jemand die "~mission(Item)" schnappt und nach ~mission(Destination) bringt. Die Art von Freude, die sich gut bezahlt macht. ~mission(Timed)~mission(Danger)
```

Hört sich glaube besser an